### PR TITLE
add environment for nextcloud

### PR DESCRIPTION
**Pull request by korelstar**
This makes ESlint to know global variables defined by Nextcloud.

### CODE REVIEW

File: lib/environments/nextcloud.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+module.exports = {
+	globals: {
+		$: 'readonly',
```
Comment (ChristophWurst) [inline lib/environments/nextcloud.js]: I don't think this is documented officially yet but we are trying really hard to get rid of these global libraries as they cause update issues. This is one of the reasons why we can't have the latest jquery in server right now as it breaks so many app :cry:

File: lib/environments/nextcloud.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+		t: 'readonly',
+		n: 'readonly',
+		OC: 'readonly',
+		OCA: 'readonly',
```
Comment (ChristophWurst) [inline lib/environments/nextcloud.js]: OCP is missing :wink:

File: lib/environments/nextcloud.js (hunk 1 above)
Comment (korelstar) [reply lib/environments/nextcloud.js]: okay, I've removed it

File: lib/environments/nextcloud.js (hunk 2 above)
Comment (korelstar) [reply lib/environments/nextcloud.js]: added :+1:

**Pull request merged**